insights/parsers/haproxy_cfg.py

```diff
@@ -70,11 +72,11 @@ def _parse_content(content):
                     section_dict[attr_key].append(attr_value)
                 else:
                     section_dict[attr_key] = attr_value
-    return haproxy_dict
+    return haproxy_dict, lines
 
 
 @parser(Specs.haproxy_cfg)
 class HaproxyCfg(Parser):
     """Class to parse file ``haproxy.cfg``."""
     def parse_content(self, content):
-        self.data = _parse_content(content)
+        self.data, self.lines = _parse_content(content)
```

Comment:
@jobselko would you mind to add an "Attributes:" section to this class docstring to define the two attributes `data` and `lines`? Thanks.